build/index.js

```diff
@@ -119,6 +121,8 @@ function injectExternalLinkClasses($) {
       return;
     }
     $a.addClass("external");
+    $a.attr("target", "_blank");
+    $a.attr("rel", "noopener noreferrer");
```

Comment:
Would that merge existing attributes or overwrite them?

Reply:
constructs like this override. It's unlike the `addClass` which cakes on like a `Set`.

Reply:
:crossed_fingers: that it won't lead to new bugs then. (Having a pre-existing `target` value is pretty uncommon).
Alternatively, you could read, merge, write it. A quick search shows me several `rel="nofollow"` in mdn/content for `<a>` elements.